pkg/rke/k8s_rke_system_images.go

```diff
@@ -8902,6 +8902,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled in Rancher v2.7.6
```

Comment:
So far we're heavily leaning towards releasing OOB _before_ 2.7.6

Same for other introduced versions.

```suggestion
		// Enabled out of band post v2.7.5
```

Reply:
done